### ISSUE TITLE
[HOLD #27913] [$1000] Different font sizes in inline and multi-line codeblock

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#18866]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. open a chat
2. send messages: inline codeblock and multiline codeblock

## expected results: 
same font size in both
## Actual results:
 different font size in each
Platform: chrome web desktop

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.13
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/43996225/1e653e2c-991c-465c-a24b-cd06b014ff62

https://github.com/Expensify/App/assets/43996225/9a86336c-4cc0-4c0f-b193-7fcb6343b556

**Expensify/Expensify Issue URL:**
**Issue reported by:**@chiragxarora
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1683823629262529

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01db882fab4bc53c2d</li>
        <li>Upwork Job ID: 1658661655173488640</li>
        <li>Last Price Increase: 2023-05-31</li>
    </ul>
</details>

**Comment by miljakljajic** [comment]
I'm OOO - this was assigned after I signed off. Reassigning. Thank you!

**Comment by JmillsExpensify** [comment]
Odd. I'm surprised this is the case, though to double check, @shawnborton is there a particular reason for this? 

I'm triaging in any case to keep the issue moving while I'm at a conference.

**Comment by phuchoang23** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Different font sizes in inline and multi-line codeblock

### What is the root cause of that problem?
We're explicitly setting fontSize for inline code here https://github.com/Expensify/App/blob/811c0f36fba4953af008e6643349d95c949627e4/src/styles/styles.js#L123

### What changes do you think we should make in order to solve the problem?
We should remove this [fontSize line](https://github.com/Expensify/App/blob/811c0f36fba4953af008e6643349d95c949627e4/src/styles/styles.js#L123). So it will reuse the fontSize of base here 

https://github.com/Expensify/App/blob/811c0f36fba4953af008e6643349d95c949627e4/src/styles/styles.js#L144

### What alternative solutions did you explore? (Optional)
Instead of remove, we can set [fontSize in line](https://github.com/Expensify/App/blob/811c0f36fba4953af008e6643349d95c949627e4/src/styles/styles.js#L123) to 
```
fontSize: variables.fontSizeNormal
```

So it will calculate fontSize normal in different device

**Comment by shawnborton** [comment]
I'm not sure if there was a good reason, maybe that it was hard to make the text wrap multiple lines and look good in line with normal text if the font size was 15 for inline code? Either way, I would love to see some proposals of how we could pull it off.

**Comment by phuchoang23** [comment]
@parasharrajat Yes. I have tested both of IOS and Android. Here is my results

<details>
  <summary>IOS</summary>

  - Before
![before_ios](https://github.com/Expensify/App/assets/129079323/3cd98841-19e3-4cf6-8400-549cd630daae)
  - After
![after_ios](https://github.com/Expensify/App/assets/129079323/4e92ba56-7c4d-40d5-b305-6b9af8ee1b8e)
</details>

<details>
  <summary>Android</summary>

  - Before
![before_android](https://github.com/Expensify/App/assets/129079323/eeda0d79-565b-4679-aeaa-24b13610b116)
  - After
![after_android](https://github.com/Expensify/App/assets/129079323/3dc76a42-e88f-4a43-a785-732951fe22fd)
</details>

**Comment by phuchoang23** [comment]
@parasharrajat Sure
- Before
![before](https://github.com/Expensify/App/assets/129079323/b09c02b2-7921-4fa3-a0ff-2eec63681363)
- After
![after](https://github.com/Expensify/App/assets/129079323/8e2814ba-e306-42fb-8fee-13b1473e70b0)

**Comment by JmillsExpensify** [comment]
@iwiznia what do you think? I'm cool to put it on hold.

**Comment by iwiznia** [comment]
Hmmmm no idea honestly. @parasharrajat can that PR solve this issue or you want to put it on hold till it is done because you can't test on android without it?

**Comment by JmillsExpensify** [comment]
Apologies ya'll, coming back to this issue. I agree that it doesn't make sense to move forward if we can't test our changes on Android. Adding a hold minimally for the react PR.

**Comment by JmillsExpensify** [comment]
Still on hold.

**Comment by iwiznia** [comment]
Same

**Comment by iwiznia** [comment]
Same, should we switch this to weekly @JmillsExpensify ?

**Comment by JmillsExpensify** [comment]
Yeah good call.

**Comment by JmillsExpensify** [comment]
Still on hold.

**Comment by iwiznia** [comment]
Going on sabbatical, please reassign another engineer if this goes off hold

**Comment by JmillsExpensify** [comment]
Hold

**Comment by JmillsExpensify** [comment]
Same

**Comment by JmillsExpensify** [comment]
Related PR still on hold.

**Comment by iwiznia** [comment]
Still on hold

**Comment by iwiznia** [comment]
Still on hold

**Comment by JmillsExpensify** [comment]
Still on hold

**Comment by JmillsExpensify** [comment]
Linked issue still on hold

**Comment by JmillsExpensify** [comment]
Still waiting on linked issue.

**Comment by JmillsExpensify** [comment]
Same

**Comment by iwiznia** [comment]
Seems that the PR we are holding on will be closed in 7 days due to inactivity: https://github.com/facebook/react-native/pull/37397#issuecomment-1894945095

**Comment by iwiznia** [comment]
Ah thanks for the context, updated the title

**Comment by JmillsExpensify** [comment]
Adding this to vip-vsb since the issue we're holding on is already part of that initiative.

**Comment by iwiznia** [comment]
Still holding

**Comment by iwiznia** [comment]
Still holding

**Comment by iwiznia** [comment]
Still holding

**Comment by iwiznia** [comment]
Still holding

**Comment by shawnborton** [comment]
Hmm I think I would rather just make all of the inline code and code blocks use 13px. Our mono font optically feels "bigger" than our regular font, so I think it's okay if we back it down to feel balanced with regular text. Is that possible? Can you also share screenshots of what is going on here - I am a bit out of the loop with the issue. cc @Expensify/design for visibility too.

**Comment by dannymcclain** [comment]
Totally agree Shawn, I think that sounds like a good course of action here.

**Comment by dubielzyk-expensify** [comment]
> Our mono font optically feels "bigger" than our regular font

Especially agree with the point above

**Comment by JmillsExpensify** [comment]
Did we go with Shawn's solution yet?

**Comment by JmillsExpensify** [comment]
Payment summary:

- Bug report: $250 @chiragxarora via Upwork
- Reviewer: $250 @parasharrajat via NewDot.

**Comment by chiragxarora** [comment]
Hello @JmillsExpensify 
I could not find any active contract on upwork for this one. Could you please send it again?
Thankyou so much

**Comment by JmillsExpensify** [comment]
No worries. @parasharrajat please make sure that we have a regression test in place before requesting payment. Thanks all!

**Comment by garrettmknight** [comment]
**Payment summary:**

Bug report: $250 @chiragxarora via Upwork
Reviewer: $250 @parasharrajat via NewDot.

**Comment by JmillsExpensify** [comment]
$250 approved for @parasharrajat

**Comment by JmillsExpensify** [comment]
P.S. Created the issue for the regression test.
